library/std/src/sys/pal/unix/fs.rs

```diff
@@ -515,11 +515,16 @@ impl FileAttr {
         SystemTime::new(self.stat.st_mtime as i64, 0)
     }
 
-    #[cfg(any(target_os = "horizon", target_os = "hurd"))]
+    #[cfg(target_os = "horizon")]
     pub fn modified(&self) -> io::Result<SystemTime> {
         Ok(SystemTime::from(self.stat.st_mtim))
     }
 
+    #[cfg(target_os = "hurd")]
+    pub fn modified(&self) -> io::Result<SystemTime> {
+        SystemTime::new(self.stat.st_mtim.tv_sec as i64, self.stat.st_mtim.tv_nsec as i64)
```

Comment:
instead of adding the cfg here, can we align the impls for hurd `SystemTime` such that the old cfg works?

Reply:
Ah I see it applies to horizon too, in that case only the existing body has to be changed with no new functions, that sounds good.